examples/with-unsplash/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "with-unsplash",
+  "version": "1.0.0",
+  "license": "MIT",
+  "author": "Diego Gennaro <kdc.crow@gmail.com>",
+  "description": "Using Next.js with Unsplash API",
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "isomorphic-unfetch": "^3.0.0",
```

Comment:
Next.js automatically polyfills `fetch` in the browser and Node.js, this package shouldn't be required.

Reply:
Package removed